TF_Settings_Web/src/Components/DocsLink/DocsLink.scss

```diff
@@ -29,44 +29,35 @@
     align-items: center;
     justify-content: space-evenly;
 
-    .qr-code--container {
-        display: flex;
-        flex-direction: row;
-        justify-content: center;
-        align-items: center;
-
-        height: 70%;
-        width: 100%;
-
-        .qr-img--container {
-            border: 20px solid white;
-            border-radius: 10px;
-            height: 95%;
-            margin-left: 5%;
-            aspect-ratio: 1/1;
-
-            > svg {
-                pointer-events: none;
-            }
-        }
+    &__qr-code {
+        border: 10px solid white;
+        margin-top: 2%;
+        border-radius: 10px;
+        height: 45%;
+        margin-left: 5%;
```

Comment:
🙈 This line causes the QR code to be off centre

Reply:
Ah good catch - was left over code from the previous style